docs/dasharo-menu-docs/overview.md

```diff
@@ -15,5 +15,133 @@ can see checkbox results on Dasharo firmware.
+    - [Device Manager](device-manager.md)
+    - [Dasharo System Features](dasharo-system-features.md)
+    - [One Time Boot](#one-time-boot)
+    - [Boot Maintenance Manager](boot-maintenance-mgr.md)
+
+NOTE: Some sections may still be under construction.
+
+* [Boot Manager Menu](#boot-manager-menu) - entered with a different key than
+  used for setup application. Lists all bootable options and allows one to
+  override the boot path.
+
+### Main Page
+
+![](/images/menus/main_page.jpeg)
+
+The page is the main view of the firmware setup application. It contains the
+board model (`MS-7D25`), installed CPU and firmware version in the top-left
+corner. In the top-right corner the CPU frequency and system RAM amount is
+shown.
+
+From the main page one may access all menus and submenus available in the
+firmware setup. Besides the menus there is also an option to:
+
+1. Change the language (currently only English is supported)
+2. `Continue` - execute the top first boot order priority
+3. `Reset` - resets the platform.
+
+The currently available menus/submenus are as follows:
+
+* [User Password Management](user-passwd-mgmt.md) - allows to set firmware
+  setup password
+* [Device Manager](device-manager.md) - allows configuring various devices and
+  features like: UEFI Secure Boot, TPM device, SATA and TCG OPAL password, etc.
+  It may also contain informational menus like Driver Health Manager, Network
+  Device List and others.
+* [Dasharo System Features](dasharo-system-features.md)- contains submenus for
+  features specific to Dasharo products and Dasharo supported platforms
+* [One Time Boot](#one-time-boot) - allows to choose which boot entry to
+  execute. It simply lists all available boot options and allows to select one
+  the same way as [Boot Manager Menu](#boot-manager-menu)
+* [Boot Maintenance Manager](boot-maintenance-mgr.md) - allows to manipulate
+  various UEFI standard variables responsible for console and boot options. One
+  may choose which devices should be used for input and output, choose to boot
+  an arbitrary file or modify the boot options and boot order.
+
+NOTE: not all submenus may be available on your platform.
+[Contact Dasharo Team](mailto:contact@dasharo.com) for more information and
+possible feature extension of your platform.
+
+### Boot Manager Menu
+
+Boot Manager Menu is an application that lists all bootable options and allows
+one to override the boot path.
+
+Boot Manager Menu is entered with a different key than setup application. It
+may be customized on your platform. The right key to use is always printed on
+the screen in the top-left corner, for example `F11`:
+
+```txt
+DEL   to enter Setup
+F11   to enter Boot Manager Menu
+ENTER to boot directly
+```
+
+After pressing the right hotkey for Boot Manager Menu, a window should pop up:
+
+![](/images/menus/boot_manager.jpeg)
+
+One may use up and down arrows to move to desired position. Pressing enter will
```

Comment:
Instead of explaining how to move in specific section, we could add one dedicated section for navigation, like here: https://www.dell.com/support/manuals/pl-pl/precision-15-7520-laptop/prec7520_om_pub/navigation-keys?guid=guid-f448dd64-1b07-4857-a446-42e313a39cc8&lang=en-us

Reply:
Navigation is also explaiend on the picture. So maybe I am overrepeating myself...

Reply:
Removed it.